client/src/components/related/Card.jsx

```diff
@@ -37,6 +40,11 @@ export default function Card({ product }) {
     return stylesResults[defaultStyle].original_price;
   };
 
+  const showComparison = () => {
+    setModal(!modal)
+    console.log('clicked')
```

Comment:
don't forget to remove console.log here later

Reply:
![image](https://user-images.githubusercontent.com/47836948/121306506-ea04d180-c8cc-11eb-8107-aeb0c528fb73.png)

Reply:
Thank you! I will remove.